docusaurus/docs/fetch.md

```diff
@@ -0,0 +1,30 @@
+---
+id: fetch
+title: '@idearium/fetch'
+---
+
+Idearium fetch wrapper.
+
+## Installation
+
+```shell
+$ yarn add -E @idearium/fetch
+```
+
+### Beta installation
+
+If you need to install a beta version, you can:
+
+```shell
+$ yarn add -E @idearium/fetch@beta
+```
+
+## Usage
+
+To use `@idearium/fetch`, simply require it at the top of your file.
+
+It is simply a fetch wrapper that automatically parses the body response into a property `result`.
+
+```js
+const { ...response, result } = await fetch('...', { fetchOpts });
```

Comment:
That isn't how you use it is it? Shouldn't it be more like...

```suggestion
const { ok, status, result } = await fetch('...', { fetchOpts });
```

Reply:
@smebberson Yeah, I was trying to highlight `result` is a custom property we add and the rest are just fetch response properties.